test/xpu_api/numerics/complex.number/complex.member.ops/divide_equal_complex.pass.cpp

```diff
@@ -21,28 +22,27 @@ test()
     assert(c.real() == -4);
     assert(c.imag() == 7.5);
     c /= c2;
-    assert(c.real() == 1.5);
-    assert(c.imag() == 2.5);
+    is_about(c.real(), (T)1.5);
```

Comment:
`explicitly cast (T)` looks not good in any case. why do we that here and everywhere below?

Reply:
Fixed.
In any case it's required to define which ```is_about``` function we should call: with ```float```, ```double``` or ```long double``` as second param.